src/synth/drumModules/snareDrum.js

```diff
@@ -32,7 +32,7 @@ export default function (audioCtx, destination, time, { level, tone, snappy }) {
   outputVCA.amplitude.value = outputLevel;
 
   // envelopes
-  const noiseEnv = new ADGenerator(LINEAR, 0.1, 75, 0, 0.5);
+  const noiseEnv = new ADGenerator(LINEAR, 0.1, 160, 0, 0.9);
```

Comment:
I really think that this is too long of a decay time. The decay time on the snare is documented to be 60ms, and I had just nudged it a bit up to 75ms to account for the differences in the envelope curves.

Reply:
Hey it's been a while, were any of the ideas here any help?